### ISSUE TITLE
Crash when provisioning GCP VPS

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-google#585]
_This issue was originally opened by @lpil as hashicorp/terraform#16313. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

When attempting a "hello world" with terraform and GCP terraform crashed.

### Terraform Version
v0.10.7

### Terraform Configuration Files
```hcl
provider "google" {
  project = "hc-louis-infra"
  region  = "europe-west2"
}

resource "google_compute_instance" "default" {
  name         = "test"
  machine_type = "f1-micro"
  zone         = "europe-west2-a"

  tags = ["louis-test"]

  boot_disk {
    initialize_params {
      image = "debian-cloud/debian-8"
    }
  }

  // Local SSD disk
  scratch_disk {
  }

  network_interface {
    network = "default"

    access_config {
      // Ephemeral IP
    }
  }
}

```

### Crash Output

[crash.log](https://github.com/hashicorp/terraform/files/1375998/crash.log)

### Expected Behavior

A VPS to be created

### Actual Behavior

Terraform crashed

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`

**Comment by rosbo** [comment]
Hi @lpil,

This issue was fixed in #556. We released a new version of the google provider yesterday. Make sure to update your provider to the latest version (1.1.0) and it should fix your issue. 

If the issue persist, please reopen the issue. 

Thank you

**Comment by lpil** [comment]
Thank you
